CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 *no unreleased changes*
 
+## 3.4.0
+
+- Allows value of `token_type` to be case-insensitive
```

Comment:
Maybe:

> ... to have any casing

?

A string can't be case-insensitive... I can compare it in a case-insensitive way, though...

Reply:
Good point—I felt like the wording was a little awkward, and now I know why. Fixed!